README.md

```diff
@@ -37,3 +37,41 @@ docker run --rm -v .:/app --workdir /app node:18 yarn install
 docker run --rm -v .:/app --workdir /app node:18 yarn build
 rm -rf node_modules
 ```
+
+## Frontend config
+
+### How to configure
+
+Following parameters is possible to configure from the `screen` -> `variant` field in the administration.
+
+### Parameters
+
+| Name                | Accepts       | Example |
+|---------------------|-------------|---------|
+| Tiles |
+| showIcons             | true,false    | `"showIcons":false` |
+| showBorders           | true,false    | `"showBorders":false` |
+| randomTiles           | true,false    | `"randomTiles":false` |
+| Expose |
+| gridExpose            | 1-9           | `"gridExpose":2` |
+| exposeShowBorder      | true,false    | `"exposeShowBorder":false` |
+| exposeShowIcon        | true,false    | `"exposeShowIcon":false` |
+| exposeTimeout         | 1-9           | `"exposeTimeout":7` |
+| exposeFontSize        | xs,s,m,l,xl   | `"exposeFontSize":"m"` |
+| Logo |
+| mosaicLogo            | true,false    | `"exposeShowIcon":false` |
+| Footer |
+| footerHeight          | 1-9           | `"footerHeight":"1"` |
+| footerImageSrc        | Url           | `"https://pathtoimagesource.io/footerimage.svg"` |
+| footerBackgroundColor | Hex format    | `"footerBackgroundColor":"#F4DCEA"` |
+| Cta box |
+| ctaBoxTitle           | Text          | `"ctaBoxTitle":"Del dit billede"` |
+| ctaBoxDescription     | Text          | `"ctaBoxDescription":"Skan koden og indsend dit bidrag"` |
+| ctaBoxImage           | Url           | `"https://pathtoimagesource.io/ctaimage.svg"` |
```

Comment:
"ctaBoxImage":"https://pathtoimagesource.io/ctaimage.svg"

Reply:
Not sure what you mean? The example includes both. Although it is different paths.